loadbalancing.md

```diff
@@ -0,0 +1,345 @@
+---
+title: Loadbalancing
+summary: How to use loadbalancing with TiDB
+---
+
+# Loadbalancing
+
+Loadbalancing is used to distribute connections from applications to TiDB Server instances. This helps to distribute the load over multiple machines and (depending on the loadbalancing option selected) can automatically re-route connections should a TiDB instance become unavailable.
+
+## Loadbalancing types
+
+The first type of loadbalancing that is available is *connector based loadbalancing*. Many of the MySQL connectors like MySQL Connector/J and MySQL Connector/C++. The benefits from this are that there is no extra network hop and the application has more information about which server it is connected to. The drawback of this is that there is no central administration and when changing the configuration you have to do this on all your application hosts. Depending on the programming language you are using the loadbalancing may not be available or not offer more advanced options. This can work with most third-party applications as this is often configured in the connection string (JDBC url for Java for example).
+
+It is also possible to use DNS round-robin. However this is not advisable as this won't prevent connections from going to a machine that is unavailable. This means that your application may have to re-try connections more often if not all servers are available.
```

Comment:
If this is not advisable, then we should not document it here.

Reply:
As DNS-RR is a common practice I think we should explain here why other solutions are better.

@morgo what's your take on this?

Reply:
I agree with @dveeden , it's very common and users need to know why we don't recommend it.

Reply:
OK, let's keep it.